packages/edit-site/src/components/global-styles/font-sizes/font-size.js

```diff
@@ -125,6 +121,18 @@ function FontSize() {
 		setIsRenameDialogOpen( ! isRenameDialogOpen );
 	};
 
+	// Navigate to the font sizes list if the font size is not available.
+	useEffect( () => {
+		if ( ! fontSize ) {
+			goTo( '/typography/font-sizes/' );
```

Comment:
could this be `goBack()` instead?

Reply:
goBack doesn't work as expected here because this runs more than once, and it ends up going back more than one screen

Reply:
Got it. Ideally we'd fix it so that it runs only once (since calling `goTo` many times is also not great), but I understand the urgency of the fix.

Could we at least specify the `isBack` option?

```suggestion
			goTo( '/typography/font-sizes/', { isBack: true } );
```

Reply:
Sorry I just saw this before getting another PR up. I can close mine, but do we need a useEffect here?

What about just returning early if the navigator params are not available?

https://github.com/WordPress/gutenberg/pull/65811/files#diff-88536b41b2c498208a524ef21e82ff4796a9bd768fafa50c82ef0bbe79d67f21R49

Reply:
Yes it's needed because the useEffect handles the navigation when the global styles are reset. In that case returning null will result in a blank screen in that sidebar.

Reply:
@matiasbenedetto is right. However, the code @ramonjd proposed is necessary to solve a different problem, and that problem also occurs in the shadow panel.

I plan to merge #65765 after this PR is shipped.

Reply:
Thanks for clarifying folks.